feeder/feeder.py

```diff
@@ -300,7 +302,7 @@ def validate_production(obj, country_code):
         raise Exception("Data from %s can't be in the future" % country_code)
     if obj.get('production', {}).get('unknown', None) is None and \
         obj.get('production', {}).get('coal', None) is None and \
-        country_code not in ['CH', 'NO']:
+        country_code not in ['CH', 'NO', 'IS']:
```

Comment:
Please remove from here. In principle, because you have "unknown" not None, this should not be a problem.